docs/current_docs/api/chaining.mdx

```diff
@@ -158,11 +158,11 @@ The example below uploads the entire current directory to the container filesyst
 :::
 
 ```shell
-dagger -m github.com/shykes/daggerverse/wolfi@v0.1.4 call container with-directory --path=/src --directory=. with-exec --args="ls","/src" stdout
+dagger -m github.com/dagger/dagger/modules/wolfi@v0.16.2 -c 'container | with-directory /src . | with-exec ls,/src | stdout'
 ```
 
-Here is an example of passing a host file to a container builder Dagger Function by chaining a call to the `Container.withFile()` function:
+Here is an example of passing a `README.md` file from the host to a container builder Dagger Function, by chaining a call to the `Container.withFile()` function:
 
 ```shell
-dagger -m github.com/shykes/daggerverse/wolfi@v0.1.4 call container with-file --path=/README.md --source=./README.md with-exec --args="cat","/README.md" stdout
+dagger -m github.com/dagger/dagger/modules/wolfi@v0.16.2 -c 'container | with-file /README.md ./README.md | with-exec cat,/README.md | stdout'
```

Comment:
This behaves differently from the original: instead of passing a host file, it passes the file from the module's Github repo. Is this expected?

Reply:
Yes, paths in shell are relative to the workdir inside the shell (you can change it with `.cd`).

Since you're loading a remote git context, to get the file from the host, you need the `host` command:

```shell
container | with-file /README.md $(host | file ./README.md) | with-exec cat /README.md | stdout
```

To be clear, if the loaded module is local, or if no module is loaded, the file will come from the host.